projects/hedge/index.js

```diff
@@ -0,0 +1,47 @@
+const { default: BigNumber } = require("bignumber.js");
+const { getCoingeckoId } = require("../helper/solana");
+const { fetchURL } = require('../helper/utils')
+
+const HEDGE_API_URL = 'http://www.hedge.so/api/overview?mode=coingecko'
+
+async function tvl() {
+    const data = (await fetchURL(HEDGE_API_URL))?.data;
+    const getCgId = await getCoingeckoId();
+    
+    const tvl = {
+        'usd-coin': new BigNumber(0),
+        'hedge-usd': new BigNumber(0),
+    };
+
+    //collateral
+    for (const mint in data?.collateralHeld) {
+        const cgId = await getCgId(mint);
+        tvl[cgId] = new BigNumber(data.collateralHeld[mint])
+    }
+
+    //psm
+    tvl['usd-coin'] = tvl['usd-coin'].plus(data?.psm?.deposited)
+
+    //stability pool
+    tvl['hedge-usd'] = tvl['hedge-usd'].plus(data?.liquidationPoolDeposits)
```

Comment:
the TVL in line 26 counts the assets that have been deposited into our Stability Pool. The assets there are used for liquidations, i.e. when a liquidation occurs, `hedge-usd` from the Stability Pool (aka liquidationPool) gets burned in return for the collateral. We incentivize users to stake into the Stability Pool by emitting our token (see https://hedge.so/pools). 

Line 17-20 only counts collateral and does not include the `hedge-usd` in the Stability Pool. Currently `hedge-usd` is not used as collateral (i.e. `tvl['hedge-usd'] === 0' before line 26), but I'm calculating it in this way in case we add some form of `hedge-usd` as collateral in the future (e.g. a LP token)

Reply:
oh I see, should I indicate that it is double counted or remove it then?

Reply:
@Haoqian-Zheng sorry, it has to be removed, I would check hubble and fix it as well